openwisp_controller/connection/tests/test_notifications.py

```diff
@@ -74,8 +56,8 @@ def test_connection_working_notification(self):
             exp_type='connection_is_working',
             exp_verb='working',
             exp_message=(
-                '<p>(SSH) connection to device <a href="{target_link}">'
-                '{n.target}</a> is {n.verb}. </p>'
+                '(SSH) connection to device <a href="{target_link}">'
+                '{n.target}</a> is {n.verb}.'
```

Comment:
Anchor tags are kept because the notification message described in register_notification_type below adds a link using markdown. 

https://github.com/openwisp/openwisp-controller/blob/4af4d01064a60024cec817c77f1870d7e73490e6/openwisp_controller/connection/apps.py#L123-L157